templates/security/esm.html

```diff
@@ -126,6 +141,19 @@ <h2>What's covered?</h2>
     <hr class="p-separator">
   </div>
 
+  <div class="row">
+    <div class="col-8">
+      <h2>ESM on Azure, AWS and Google Cloud with Ubuntu Pro</h2>
+      <p>
+        If you are running Ubuntu 16.04 LTS images on the public cloud and are looking for continued security coverage with ESM, it is recommended to launch new, Ubuntu Pro images for Azure, AWS and Google Cloud (GCP coming soon). Ubuntu Pro images are paid, premium images that are optimised and priced for the cloud, with security and compliance features built in.
```

Comment:
The Azure, AWS and Google cloud in the paragraph are also links in the copy doc

Reply:
Added the links.